Octokit.Tests/Reactive/ObservableIssueCommentsClientTests.cs

```diff
@@ -104,38 +152,73 @@ public void RequestsCorrectUrl()
             }
 
             [Fact]
+            public void RequestsCorrectUrlWithRepositoryId()
+            {
+                var gitHubClient = Substitute.For<IGitHubClient>();
+                var client = new ObservableIssueCommentsClient(gitHubClient);
+
+                client.GetAllForIssue(1, 3);
+
+                gitHubClient.Connection.Received(1).Get<List<IssueComment>>(
+                    new Uri("repositories/1/issues/3/comments", UriKind.Relative), Args.EmptyDictionary, null);
+            }
+
+            [Fact]
             public void RequestsCorrectUrlWithApiOptions()
             {
                 var gitHubClient = Substitute.For<IGitHubClient>();
                 var client = new ObservableIssueCommentsClient(gitHubClient);
 
-                var options=new ApiOptions
+                var options = new ApiOptions
                 {
                     StartPage = 1,
                     PageSize = 1,
                     PageCount = 1
                 };
+
                 client.GetAllForIssue("fake", "repo", 3, options);
 
                 gitHubClient.Connection.Received(1).Get<List<IssueComment>>(
-                    new Uri("repos/fake/repo/issues/3/comments", UriKind.Relative), 
-                    Arg.Any<Dictionary<string, string>>(),
-                    "application/vnd.github.squirrel-girl-preview");
+                    new Uri("repos/fake/repo/issues/3/comments", UriKind.Relative), Arg.Is<IDictionary<string, string>>(d => d.Count == 2), null);
```

Comment:
I think you need to bring back the custom `Accept` header here to get that test passing...